admin/class-admin-init.php

```diff
@@ -61,6 +62,49 @@ public function __construct() {
 	}
 
 	/**
+	 * Validates the extensions and show a notice for the invalid extensions.
+	 */
+	public function validate_extensions() {
+
+		if ( filter_input( INPUT_GET, 'page' ) === WPSEO_Admin::PAGE_IDENTIFIER ) {
+			apply_filters( 'yoast-active-extensions', array() );
```

Comment:
If I'm missing something, please add the context in the comments to clarify for future developers.